### ISSUE TITLE
Use Jinja for templates

### ISSUE THREAD

**Issue by janek-warchol** [janek-warchol/selenized#21]
This should help shorten them (by using jinja's for construct).

**Comment by dngray** [comment]
Yes this would be good. I've used Jinja a fair bit with [Yadm](https://yadm.io) dotfile manager. An example in a [ssh_config](https://github.com/thingfox/dotfiles/blob/master/.ssh/config.tpl#L20).

For testing you could use [envtpl](https://github.com/andreasjansson/envtpl), though I think that [might be unmaintained](https://github.com/TheLocehiliosan/yadm/issues/159) in favor of [j2cli](https://github.com/kolypto/j2cli)
